--- Aegisub-3.2.2.orig/tools/Makefile	2016-09-09 15:06:52.437581247 +0200
+++ Aegisub-3.2.2.new/tools/Makefile	2016-09-09 15:07:21.562160809 +0200
@@ -7,7 +7,7 @@
 endif
 
 repack-thes-dict_OBJ  := $(d)repack-thes-dict.o $(TOP)lib/libaegisub.a
-repack-thes-dict_LIBS := $(LIBS_BOOST) $(LIBS_ICU)
+repack-thes-dict_LIBS := $(LIBS_BOOST) $(LIBS_ICU) -lpthread
 repack-thes-dict_CPPFLAGS := -I$(TOP) -I$(TOP)libaegisub/include $(CFLAGS_ICU)
 
 PROGRAM += $(d)repack-thes-dict
